.cargo/config.toml

```diff
@@ -0,0 +1,5 @@
+[build]
+# Without this flag, when linking static libruby, the linker removes symbols
+# (such as `_rb_ext_ractor_safe`) which it thinks are dead code... but they are
+# not, and they need to be included for the `embed` feature to work.
+rustflags = ["-C", "link-dead-code=on"]
```

Comment:
I was shocked to run across https://zenn.dev/ahogappa/articles/4be34946fd8b65, it's rare that I stumble into an obscure Rust+Ruby static linking bug.

Reply:
I think .cargo/config.toml only applies when directly building a crate, not when it's a dependency. So this wouldn't  apply for anyone depending on Magnus, only when developing Magnus, but it's not needed when developing Magnus so it's redundant.

It's probably worth calling out in the readme though.